### DISCUSSION TITLE
New IKEA bulb is not supported

### DISCUSSION THREAD

**Discussion by petkov** [Koenkk/zigbee2mqtt#12384] [Devices]
Having trouble adding new IKEA 800lm bulb to zigbee2mqtt. Model 404.391.69 or LED1924G9
Error message I got after pairing: Received message from unsupported device with Zigbee model 'TRADFRI bulb E26 CWS 800lm' and manufacturer name 'IKEA of Sweden'
Looks like it is similar to https://github.com/Koenkk/zigbee2mqtt/discussions/7215

Any help appreciated

**Comment by Koenkk** [comment]
Should be supported in z2m 1.25.1

**Reply by petkov** [reply]
Strange, my z2m version in HA is 1.18.1-1, which is 1 year old. Auto update is turned on, so I don't understand why it doesn't get new versions.

**Reply by petkov** [reply]
It turned out I used old repository, once I've uninstalled z2m and updated repository I started seeing z2m with new version. Installing it right now.

**Reply by petkov** [reply]
Everything works now!
